modules/nodes-scheduler-default-creating.adoc

```diff
@@ -57,10 +56,10 @@ To create the scheduler policy:
 <1> Add the predicates as needed.
 <2> Add the priorities as needed.
 
-. Create a ConfigMap based on the JSON file:
+. Create a ConfigMap based on the scheduler JSON file:
 +
 ----
-$ oc create configmap -n openshift-config --from-file=policy.cfg <configmap-name>
+$ oc create configmap -n openshift-config --from-file=<policy-json-file> <configmap-name>
```

Comment:
The policy file name should be policy.cfg so that key in configmap is `policy.cfg` without that, the scheduler will fail to pick up the config because of limitation in kube-scheduler code.

Copying from kube-scheduler documentation:

```  The config must be provided as the value of an element in 'Data' map with the key='policy.cfg'```